engine/render/renderer/renderer/Common.h

```diff
@@ -26,4 +20,4 @@ struct CameraData
 };
```

Comment:
Realistically I'd say this is more just `CameraData.cpp` than anything. "Common" is not all that descriptive.

Reply:
I was thinking of storing more common structs here, but until we have them I think this makes more sense as a change.